x-pack/packages/kbn-slo-schema/src/rest_specs/slo.ts

```diff
@@ -134,6 +134,31 @@ const getSLODiagnosisParamsSchema = t.type({
   path: t.type({ id: t.string }),
 });
 
+const getSLOBurnRatesResponseSchema = t.type({
+  burnRates: t.array(
+    t.type({
+      name: t.string,
+      burnRate: t.number,
+      sli: t.number,
+    })
+  ),
+});
+
+const getSLOBurnRatesParamsSchema = t.type({
+  path: t.type({ id: t.string }),
+  body: t.type({
+    windows: t.array(
+      t.type({
+        name: t.string,
+        duration: t.type({
+          value: t.number,
+          unit: t.string,
+        }),
```

Comment:
🍰 You probably can reuse the `durationType` for the duration field. It parses and validates the duration